sktime/forecasting/compose/_skforecast_reduce.py

```diff
@@ -0,0 +1,380 @@
+        alpha : list of float (guaranteed not None and floats in [0,1] interval)
+            A list of probabilities at which quantile forecasts are computed.
+
+        Returns
+        -------
+        quantiles : pd.DataFrame
+            Column has multi-index: first level is variable name from y in fit,
+                second level being the values of alpha passed to the function.
+            Row index is fh, with additional (upper) levels equal to instance levels,
+                    from y seen in fit, if y_inner_mtype is Panel or Hierarchical.
+            Entries are quantile forecasts, for var in col index,
+                at quantile probability in second col index, for the row index.
+        """
+        (
+            maximum_forecast_horizon,
+            absolute_horizons,
+            horizon_positions,
+        ) = self._get_horizon_details(fh)
+
+        var_names = list(map(str, self._get_varnames()))
+        var_name = var_names[0]
+
+        quantile_predictions_indices = pandas.MultiIndex.from_product(
+            [var_names, alpha]
+        )
+        quantile_predictions = pandas.DataFrame(
+            index=absolute_horizons, columns=quantile_predictions_indices
+        )
+
+        bootstrap_predictions = self._forecaster.predict_bootstrapping(
+            maximum_forecast_horizon, exog=self._coerce_column_names(X)
+        )
+        bootstrap_quantiles = bootstrap_predictions.quantile(
+            q=alpha, axis=1
+        ).transpose()
+
+        for quantile in alpha:
+            quantile_predictions[(var_name, quantile)] = bootstrap_quantiles.to_numpy()[
+                horizon_positions
+            ]
+
+        return quantile_predictions
+
+    @classmethod
+    def get_test_params(cls, parameter_set="default"):
+        """Return testing parameter settings for the estimator.
+
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return, for use in tests. If no
+            special parameters are defined for a value, will return `"default"` set.
+            There are currently no reserved values for forecasters.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, i.e.,
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`
+        """
+        del parameter_set  # to avoid being detected as unused by ``vulture`` etc.
+
+        from sklearn.ensemble import RandomForestRegressor
+        from sklearn.linear_model import LinearRegression
+
+        params = [
+            {"regressor": LinearRegression(), "lags": 2},
+            {"regressor": RandomForestRegressor(), "lags": [1, 3]},
```

Comment:
We should probably add coverage for more arguments.

Reply:
Will do once the existing errors are resolved.

Reply:
well, they now are :-)